bot/src/bounded-contexts/analytics/domain/services/processors/RatioDataProcessor.ts

```
@@ -0,0 +1,177 @@
+      const groupName = dataPoint.groupName || 'Unknown';
+      
+      if (!groupStats.has(groupName)) {
+        groupStats.set(groupName, {
+          kills: 0,
+          losses: 0,
+          killValue: 0,
+          lossValue: 0
+        });
+      }
+      
+      const stats = groupStats.get(groupName)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)